### PR TITLE
Update Jenkins-Proxy

### PR DESCRIPTION
**Pull request by sthaha**
This update addresses 504 gateway error that is shown when Jenkins
takes too long to boot.

### CODE REVIEW

File: dsaas-services/f8-jenkins-idler.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ services:
   - name: staging
     parameters:
       IMAGE: quay.io/openshiftio/rhel-fabric8-services-fabric8-jenkins-idler
-- hash: a6e9c5c1f9ba7cd6417f9e020cddf49644e16d67
+- hash: 99399c90d426140e31f442ab1c34eb77cc58471c
```
Comment (sthaha) [inline dsaas-services/f8-jenkins-idler.yaml]: https://github.com/fabric8-services/fabric8-jenkins-proxy/commit/99399c90d426140e31f442ab1c34eb77cc58471c

**Comment by ppitonak** [comment]
[test]

**Comment by sthaha** [comment]
[test]

**Comment by sthaha** [comment]
Failure doesn't seem related to Jenkins-Proxy 
```
[11:22:18.839]: Saved page source to: target/screenshots/07-01-afterEach.html
F    ✗ dashboard (2 secs)
      - Failed: No element found using locator: by.cssContainingText("b", "Total:")
          at elementArrayFinder.getWebElements.then (/opt/fabric8-test/node_modules/protractor/built/element.js:814:27)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)Error
          at ElementArrayFinder.applyAction_ (/opt/fabric8-test/node_modules/protractor/built/element.js:459:27)
          at ElementArrayFinder.(anonymous function).args [as getText] (/opt/fabric8-test/node_modules/protractor/built/element.js:91:29)
          at ElementFinder.(anonymous function).args [as getText] (/opt/fabric8-test/node_modules/protractor/built/element.js:831:22)
          at AnalyticsCard.getTotalDependenciesCount (/opt/fabric8-test/src/page_objects/space_dashboard.page.ts:206:72)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
      From asynchronous test: 
      Error
          at Suite.describe (/opt/fabric8-test/src/smoke.spec.ts:107:3)
          at Env.wrapper [as describe] (/opt/fabric8-test/node_modules/jasmine-fail-fast/node_modules/lodash/index.js:3592:19)
          at Object.<anonymous> (/opt/fabric8-test/src/smoke.spec.ts:16:1)
          at Module._compile (module.js:652:30)
          at Object.Module._extensions..js (module.js:663:10)
[11:22:18.843]: --- After all ---
```

**Comment by ppitonak** [comment]
@sthaha, correct, this is a known issue https://github.com/openshiftio/openshift.io/issues/3744

**Comment by sthaha** [comment]
@ppitonak does this mean updates are blocked until that issue is resolved? Or can we merge the patch?

**Comment by ppitonak** [comment]
@sthaha we don't enforce E2E smoke test checks until they are stable, feel free to merge if you feel that the change won't break production

**Pull request merged**